packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3303,6 +3304,11 @@ export default class Meeting extends StatelessWebexPlugin {
         }) &&
           this.meetingInfo?.video?.supportHDV) ||
         !this.arePolicyRestrictionsSupported(),
+      enforceVirtualBackground:
+        ControlsOptionsUtil.hasPolicies({
+          requiredPolicies: [SELF_POLICY.ENFORCE_VIRTUAL_BACKGROUND],
+          policies: this.selfUserPolicies,
+        }) && this.arePolicyRestrictionsSupported(),
```

Comment:
all other meeting actions have the the conditions for `this.arePolicyRestrictionsSupported()` differently (negated). 
I don't have much context, so just pointing it out.